src/index.ts

```diff
@@ -50,4 +51,29 @@ export default {
     )
     return data
   },
+
+  /**
+   * processScan
+   * @param type type of scan performed
+   * @param filename filename of scan results
+   */
+  async processScan(type: String, filename: String) {
+    assert(this.processScanEndpoint, 'process scan endpoint is not available')
+    assert(this.phalanxApiKey, 'Phalanx API key is not available')
+
+    const { data } = await axios.post(
+      this.processScanEndpoint,
+      { 
+        type: type, 
+        filename: filename, 
```

Comment:
@wrightmalone Maybe we use file upload id here instead of name? I think name will also always be unique, but I think it feels better to use the `id` field from the `file_uploads` table where the file will be referenced.

Reply:
works for me, done